variables.tf

```diff
@@ -126,7 +126,7 @@ variable "remove_default_node_pool" {
 
 variable "disable_legacy_metadata_endpoints" {
   description = "Disable the /0.1/ and /v1beta1/ metadata server endpoints on the node. Changing this value will cause all node pools to be recreated."
-  default     = true
+  default     = "true"
```

Comment:
This change is not relevant to the branch topic.

Reply:
This is also the result of a skew between the templates and autogen'd code in previous PR: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/commit/6813253b5f314a4282b231ce1489f0d65180e1d9

The diff is unavoidable when generating the documentation.